ztp/source-crs/MachineConfigContainerMountNS.yaml

```diff
@@ -66,16 +64,12 @@ spec:
             ExecStart=
             ExecStart=bash -c "nsenter --mount=%t/container-mount-namespace/mnt \
                 ${ORIG_EXECSTART} --housekeeping-interval=30s"
-        # The kubelet service config is included with the container
-        # mount namespace because the override of the ExecStart for
-        # kubelet can only be done once (cannot accumulate changes
-        # across multiple drop-ins).
-        # Defaults:
-        #  Max Housekeeping : 15s
-        #  Housekeeping : 10s
-        #  Eviction : 10s
-        - name: 30-kubelet-interval-tuning.conf
-          contents: |
+          name: 20-container-mount-namespace.conf
+        name: kubelet.service
+      - dropins:
```

Comment:
This 2nd drop-in for the same service may be problematic.  Need to test IRL first, or wait for the tool upgrade in https://github.com/openshift-kni/cnf-features-deploy/pull/757

Reply:
/hold

Reply:
Confirmed: This code is DOA.  We need to wait for #757 to merge first to fix the tool that does the MC object generation, and then I'll update this again.

Reply:
/unhold